inference-engine/thirdparty/clDNN/tests/test_cases/eltwise_gpu_test.cpp

```diff
@@ -3193,6 +3193,174 @@ TEST(DISABLED_eltwise_gpu, generic_random) {
     }
 }
 
+//
+struct eltwise_same_input_test_params {
+    data_types  input_type;
+    tensor      input_size;
+
+    format::type in_format;
+    format::type out_format;
+};
+
+struct eltwise_same_input_test : testing::TestWithParam<eltwise_same_input_test_params>
```

Comment:
This seems to be quite similar to `generic_eltwise_test`. What is the difference?

Reply:
I wanted to make a simple test that I represented the existing target bug.
Should it be supposed to be merged to generic test if it seems to be similar?